dataactcore/models/lookups.py

```diff
@@ -88,11 +88,14 @@
     LookupType(2, 'writer', 'This user is allowed to create and edit any submission for their agency'),
     LookupType(3, 'submitter', 'This user is allowed to certify and submit any submission for their agency'),
     # Placeholder 4: website_admin
-    LookupType(5, 'fabs', 'This user is allowed to create and publish any FABS data for their agency')
+    LookupType(5, 'fabs', 'This user is allowed to publish any FABS data for their agency'),
+    LookupType(6, 'editfabs', 'This user is allowed to create and edit any FABS data for their agency')
```

Comment:
I know it says 4 is a placeholder, but website admin is a flag on the user now. Also, we probably want to make the edit a lower number than the "everything" because that's the way we've been doing it in the past. We can just slide `editfabs` into the 4 slot I think.